planner/core/point_get_plan.go

```diff
@@ -329,36 +335,37 @@ func buildSchemaFromFields(ctx sessionctx.Context, dbName model.CIStr, tbl *mode
 	return schema
 }
 
-func getSingleTableName(tableRefs *ast.TableRefsClause) *ast.TableName {
+func getSingleTableNameAndAlias(tableRefs *ast.TableRefsClause) (tblName *ast.TableName, tblAlias model.CIStr) {
```

Comment:
Could you add a comment for this function? To explain that the `tblAlias` might be the true name of the table if there is no table alias in the query.

Reply:
done